llarp/util/thread/logic.cpp

```diff
@@ -90,10 +96,8 @@ namespace llarp
     }
     if(m_Thread->LooksFull(5))
     {
-      LogErrorExplicit(TAG, LINE,
-                      "holy crap, we are trying to queue a job onto the logic "
-                      "thread but "
-                      "it looks full");
+      LogErrorExplicit(TAG, LINE, "holy crap, we are trying to queue a job "
```

Comment:
ok, so since this got reformatted anyway, perhaps now is a good time to change the log message >_>